src/site/content/en/blog/install-thumbor/index.md

```diff
@@ -0,0 +1,184 @@
+---
+title: How to install the Thumbor image CDN
+subhead: Learn how to install the Thumbor image CDN on your own server.
```

Comment:
Would it be possible to punch up the subhead a bit? I've been encouraging folks to use it as an opportunity to give the reader a bit more context on why they should care about the topic.

Reply:
Is this better?

Reply:
yep I think that's good. Thanks!